src/common/Label.tsx

```diff
@@ -1,23 +1,32 @@
-import React, { useContext } from "react";
+import React from "react";
 
 import { useGlobalContext } from "../context/Context";
 import Helpers from "../lib/helpers";
 
+import { Size, GenericObject } from "../types";
+
 type Props = {
-  children: any;
+  children: React.ReactNode;
   onClick?: () => void;
-  style?: Object;
-  size?: "xs" | "sm" | "lg" | "xl";
+  style?: GenericObject;
```

Comment:
I think you can use:

```suggestion
  style?: React.CSSProperties;
```